crates/editor/src/editor.rs

```diff
@@ -1707,7 +1709,7 @@ impl CodeActionsMenu {
                                         }
                                     }),
                                 )
-                                .child(SharedString::from(task.resolved_label.clone()))
+                                .child(SharedString::from(task.resolved_label.replace("\n", "")))
```

Comment:
I'm pretty sure we have a `single_line` helper on Label, if you'd like to use that.

Reply:
It replaces `\n` with `␤` which looks weird. I'd rather prefer nothing or a space, not sure which is better